### ISSUE TITLE
Stopped working 

### ISSUE THREAD

**Issue by mihaimdinca** [TarheelGrad1998/gallery-card#86]
I am on Home Assistant 2024.1.3 and the card just returns an error message - Custom element does not exist: gallery-card.

**Comment by mihaimdinca** [comment]
I am very sorry to learn that this project is not supported anymore. I found it very useful. Unfortunately I cannot get it to work anymore even after the latest update. I get the custom element does not exist: gallery-card error message. The card was installed via HACS. Its files are installed under the /www/community/gallery-card similar to other custom cards. I tried uninstalling and reinstalling without making any difference. Anyone willing to point me in the right direction?

**Comment by mihaimdinca** [comment]
Apparently the card works but only if I access it from outside the home network

**Comment by Judufu** [comment]
what do you mean from outside the network?

**Comment by pgorod** [comment]
My gallery card stopped working today... :-(

**Comment by padrien51** [comment]
Me too, it is no working anymore whereas it was ok the previous days. Any solution?

**Comment by FalconUK** [comment]
Same for me. Has anyone found an alternative, as this isn't maintained any more?

**Comment by TarheelGrad1998** [comment]
Yeah, definitely broken.  I'll check later to see if there's an easy fix

**Comment by mihaimdinca** [comment]
https://github.com/luixal/lovelace-media-source-image-card not a plug and play replacement but depending on use case, you can use this with its templating features and two buttons and a number input for navigation between pictures. Not the same, but working well

**Comment by cvjensen** [comment]
> Hrm, I started to investigate, and mine started working again. Anyone else?
> 
> My guess is it was something with the unpkg.com dependencies. But hard for me to trouble shoot given it is working for me now.

Mine is also working again . I saw some Cors errors in the browser when it didnt work . They mentioned unpkg

**Comment by jgverweij** [comment]
Here also working again. Did a Home Assistant OS update with reboot.

**Comment by padrien51** [comment]
Yes me too. Really strange…but good news :)

**Comment by harambe88** [comment]
I'm having still trouble. What is a "hard refresh of the browser"? Just closing and restarting? 
That didn't work for me. Using another browser didn't work either.

**Comment by TarheelGrad1998** [comment]
It is basically clearing your browser cache.  I usually do it by holding shift while hitting refresh, but it varies by browser and OS.

**Comment by harambe88** [comment]
Thanks for the shift-F5 tip! I tried this and I in addition deleted all browser data under settings and also used Firefox as an alternative browser. Same result.

I uninstalled the gallery-card and reinstalled it. I also checked the paths in the lovelace resources. Nothing helped.

**Comment by lukelalo** [comment]
As this project has been discontinued, I've fork it to fix the issues and improved the ci:
https://github.com/lukelalo/gallery-card

**Comment by harambe88** [comment]
Thank you! I found the reason for the card not working. Some how the path in the resources didn't match anymore. No idea why this happened. Wasn't me :))

**Comment by TarheelGrad1998** [comment]
> As this project has been discontinued, I've fork it to fix the issues and improved the ci: https://github.com/lukelalo/gallery-card

Good luck with it.  I'll be interested to see where it can go with more focus and attention.  I've added a link to your repo in my readme.

**Comment by FalconUK** [comment]
for us mere mortals, what do we need to do to keep up to date? I guess the fork changes the identity on HACS? Do we need to remove this one and then install @lukelalo's version.

**Comment by lukelalo** [comment]
Yes, just that. The name of the component is gallery-card, so you don't need to change it on your dashboards, only install the forked version and its done

**Comment by lukelalo** [comment]
> > As this project has been discontinued, I've fork it to fix the issues and improved the ci: https://github.com/lukelalo/gallery-card
> 
> Good luck with it. I'll be interested to see where it can go with more focus and attention. I've added a link to your repo in my readme.

Thanks! I'll do my best

**Comment by FalconUK** [comment]
> Yes, just that. The name of the component is gallery-card, so you don't need to change it on your dashboards, only install the forked version and its done

The new version (pointing to your repo) doesn't show up in HACS in Home Assistant, but then I haven't removed the old version yet. How is it best to confirm that your forked version is in HACS, ready to be downloaded?

Thanks for your help.

**Comment by lukelalo** [comment]
As for now, until I find how to add it on HACS repository, you need to download my github repo and add it to HACS
